TidyTuesday (#125)
yabellini/SitioAcademico: content/blog/2024-01-06-TidyTuesday/index.md

---
title: Bring Your Own Data week for TidyTuesday
author: Yanina Bellini Saibene
summary: "It's _Bring Your Own Data_ week for _#TidyTuesday_, and I decided to analyze my records of events."
date: '2024-01-06'
categories:
  - 100DaysToOffload
  - English
tags:
  - English
  - 100DaysToOffload
  - rstats
  - TidyTuesday 
---

It's _Bring Your Own Data_ week for _#TidyTuesday_, and I decided to analyze my records of events. 

> I have the habit of recording everything because of my academic jobs. I continue to write down the different events in which I participate and some interesting information about them.  In addition to update the data with what I did during 2023, I have been cleaning up the data, completing the information and removing duplicates.

Author: ```suggestion
I have the habit of recording everything because of my academic jobs. I continue to write down the events I participate in and some interesting information about them.  For this exercise, in addition to updating the data with what I did during 2023, I have been cleaning up the data, completing the information, and removing duplicates.
```

  geom_col(fill = "#56B4E9") +
  coord_flip() +
  labs(title = "Total Events by Type",
       y = "",
       x = "") +
  theme_minimal()

```

I got this two ranked bar plots with my data by role and by type of events:

{{< figure src="bar.png" alt="One bar plot next to the other. The first one shows 5 roles, being Speaker the most important, followed by teacher.  In the second bar plot there are 13 categories, being Course the most important.  The following are different kind of talks followed by conferences, panels, exibitions, podcast, among others types." >}}

## Combined plot

Finally, I want to show all of this data together, so I use the package `patchwork` to create a combined plot with all the preview elements.  The code order each of the other plots in the patchwork: first line is all the donuts plots next to each other (p2, p4 y p5), bellow is the line plot (p1), and below it the two bar plot (p0, p6) one next to the other. 

``` r

(p2 | p4 | p5) / (p1 / (p0 | p6))

```

This is the final plot:

{{< figure src="featured.png" alt="Donuts plot on the first row, below the donuts the line plot and in thelast row the two bar plots." >}}

> I can improve the annotations for the patchwork, but I kind of like the final output. :-) 

Author: ```suggestion
I can improve the annotations for the patchwork, and the disposition of the bar plots, but I kind of like the final output. :-) 
```
